src/core/Akka.Remote.Tests.MultiNode/RemoteDeliverySpec.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Akka.Actor;
+using Akka.Remote.TestKit;
+
+namespace Akka.Remote.Tests.MultiNode
+{
+    public class RemoteDeliveryMultiNetSpec : MultiNodeConfig
+    {
+        public RemoteDeliveryMultiNetSpec()
+        {
+            First = Role("first");
+            Second = Role("second");
+            Third = Role("third");
+
+            CommonConfig = DebugConfig(false);
+        }
+
+        public RoleName First { get; private set; }
+        public RoleName Second { get; private set; }
+        public RoleName Third { get; private set; }
+
+        public sealed class Letter
+        {
+            public Letter(int n, List<IActorRef> route)
+            {
+                N = n;
+                Route = route;
+            }
+
+            public int N { get; private set; }
+            public List<IActorRef> Route { get; private set; }
+        }
+
+        public class Postman : UntypedActor
+        {
+            protected override void OnReceive(object message)
+            {
+                var letter = message as Letter;
+                if (letter != null)
+                {
+                    letter.Route[0].Tell(new Letter(letter.N, letter.Route.Skip(1).ToList()));
```

Comment:
technically `letter.Route.Skip(1)` will only work for a list of size 2 - you should call `Skip(N-1)`